src/Faker/Provider/fr_FR/Company.php

```diff
@@ -166,4 +165,16 @@ protected static function isCatchPhraseValid($catchPhrase)
 
         return true;
     }
+
+    /**
+     * @link http://www.emploi.com/emploi/index-metiers.html
```

Comment:
this content is `© Keljob.com tous droits réservés` and can't be reused in a MIT library.